build.clj

```diff
@@ -0,0 +1,86 @@
+(ns build
+  (:require
+    [borkdude.gh-release-artifact :as gh]
+    [clojure.tools.build.api :as b]
+    [deps-deploy.deps-deploy :as dd]))
+
+
+(def lib 'replikativ/datahike)
+(def version (format "0.4.%s" (b/git-count-revs nil)))
+(def current-commit (gh/current-commit))
+(def class-dir "target/classes")
+(def basis (b/create-basis {:project "deps.edn"}))
+(def jar-file (format "target/%s-%s.jar" (name lib) version))
+
+
+(defn clean
+  [_]
```

Comment:
What's the reason behind the empty parameters?

Reply:
there is a params-list passed and nil when not used